src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -27,6 +27,9 @@
 	@JsonProperty("Privileged")
 	private boolean privileged;
 
+	@JsonProperty("ReadonlyRootfs")
+	private boolean readonlyRootfs;
```

Comment:
Best will be to use object types instead of primitives. See https://github.com/docker-java/docker-java/issues/246